python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -352,6 +356,48 @@ def __new__(
         )
 
 
+@whitelist_for_serdes
+@experimental
+class BackfillRequest(
+    NamedTuple(
+        "_BackfillRequest",
+        [
+            ("asset_partitions", "AssetGraphSubset"),
+            ("tags", PublicAttr[Optional[Mapping[str, str]]]),
+        ],
+    )
+):
+    @property
+    def asset_keys(self) -> AbstractSet[AssetKey]:
+        return self.asset_partitions.asset_keys
+
+    @classmethod
+    def from_asset_selection_and_partitions(
+        cls,
+        asset_selection: Sequence[AssetKey],
+        partition_keys: Sequence[str],
```

Comment:
The UI now supports asset backfills that target only unpartitioned assets, thoughts on making `partition_keys` arg optional?

Reply:
👍